warden/app/custom-msg/custom_msg.go

```diff
@@ -0,0 +1,73 @@
+package custom_msg
+
+import (
+	"encoding/json"
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	types "github.com/warden-protocol/wardenprotocol/warden/x/warden/types/v1beta2"
+)
+
+type Addr string
+type KeyType string
+
+const (
+	KeyType_Unspecified    KeyType = "unspecified"
+	KeyType_EcdsaSecp256k1 KeyType = "ecdsa_secp256k1"
+	KeyType_EddsaEd25519   KeyType = "eddsa_ed25519"
+)
```

Comment:
Unfortunately, we can't use existing enum because here we deserializing type [declared](https://github.com/warden-protocol/wardenprotocol/blob/d07e0bb8c97c18d3fd43fb5cd4bf4d2758454ac5/contracts/packages/bindings/src/msg.rs#L14) on the contract side.